src/components/Notification/NotificationTray.react.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from "react";
+import { Notification, Dropdown } from "../";
+
+type Props = {|
+  /**
+   * Notification components
+   */
+  +children?: React.ChildrenArray<React.Element<typeof Notification>>,
```

Comment:
I like how we have a specific type for children and also the option for the shorthand object type 👍